pg-commands/_template.md

```diff
@@ -0,0 +1,152 @@
+# ORDER BY
+
+<!---
+General structure:
+* Definition of keyword or function
+* Time-series focused friendly intro
+* Simple examples 
+* More complex examples 
+-->
+
+By default, in PostgreSQL and therefore TimescaleDB, the results from a basic
+`SELECT` query are unordered. You must include the [`ORDER BY`][orderby-postgres]
+clause to guarantee the order of your results. When you use `ORDER BY`, you need
+to specify the columns to order the results on then, optionally, give the
+ordering direction, either ascending or descending. If you don't specify the
+order direction for your columns, the default ordering is ascending.
+
+When you query time-series data, you need ordered results. Time occurs linearly
```

Comment:
This feels incredibly strong.

Reply:
It's not necessary, and you don't always order things on time, it feels kinda weird to make this so strong here.